scaffold-tfmigrate/action.yaml

```diff
@@ -32,6 +32,15 @@ runs:
           sed "s|%%TARGET%%|${TFACTION_TARGET}|g" ${{ github.action_path }}/tfmigrate.hcl |
           sed 's|%%S3_BUCKET_NAME_TFMIGRATE_HISTORY%%|${{steps.target-config.outputs.s3_bucket_name_tfmigrate_history}}|g' > .tfmigrate.hcl
         fi
+      if: steps.target-config.outputs.gcs_bucket_name_tfmigrate_history == ''
```

Comment:
I'm not so sure that this `if` is acceptable or not; there is no flag nor configuration to specify which cloud vendor is used.

I believe this `if` will not affect existing users (mostly AWS guys, probably), at least.

Reply:
Sure! Thank you. I thought my proposal can keep backward compatibility, but this change is more natural and reasonable :)

Reply:
Done by 397905f, thanks for your notice!